src/client/components/MyProfile/ProfilePage/ProfileComponent.js

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState, useContext } from 'react';
+import { handlerUserContext } from '../../../App';
+import './ProfileComponent.styles.css';
+import { ProfileCardComponent } from '../../ProfileCardComponent/ProfileCardComponent';
+import { YourUploadedCVs } from '../YourUploadedCVs/YourUploadedCVs';
+import TitleCvCard from '../TitleCvCard/TitleCvCard';
+
+export default function ProfileComponent() {
+  const { setUserName } = useContext(handlerUserContext);
+  const [cvsList, setCvsList] = useState([]);
+  const [userData, setUserData] = useState([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const userId = 2;
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = await fetch('/api/users');
+        if (response.status !== 200) {
+          throw new Error('fail to connect to the Api');
+        }
+        const users = await response.json();
+        const data = users.find((user) => user.id === userId);
+        data.linkedin = data.linkedin.substring(28);
```

Comment:
any suggestion to set these values in cleaner way?

Reply:
:) 

1: Ask the backend to create a field where it's formatted the way you would like it to be. 
2: Create a new data object, with the relevant changes, i.e: 

```
let data = ... //fetching data
let formattedData = {...data, github: data.github.substring(19), // and whatever fields you need to format}
```

The 2nd is fastest, and we need fast right now i believe. But you decide. 👍